xbmc/cores/DllLoader/exports/emu_msvcrt.cpp

```diff
@@ -72,6 +72,13 @@
 #include "platform/win32/CharsetConverter.h"
 #endif
 
+#ifndef _PATH_MOUNTED
+#define _PATH_MOUNTED "/proc/mounts"
+#endif
+#ifndef _PATH_MNTTAB
+#define _PATH_MNTTAB "/etc/fstab"
+#endif
```

Comment:
On second thought, adding a `#if defined(TARGET_LINUX)` around this is probably a good idea.

Reply:
Ah, yes. Probably TARGET_POSIX makes more sense as its the common location for all POSIX systems, right? The usage of _PATH_MNTTAB and _PATH_MOUNTED are ifdefed to TARGET_LINUX anyway so it shouldn't matter either way.

Reply:
`TARGET_POSIX` would include OSX and I have no clue if it makes sense for that. I'm in favor of `TARGET_LINUX` as (currently) no other platform is using this.

Reply:
Alright, makes sense. Updated it